### PR TITLE
Add support for dotnet restore / remove dnx commands

### PR DESCRIPTION
**Pull request by caslan**
@gregg-miskelly @DustinCampbell

### CODE REVIEW

**Comment by gregg-miskelly** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -84,12 +81,7 @@
             {
                 "command": "o.restore",
```
Comment (DustinCampbell) [inline package.json]: Any reason not to lose the "o." prefix here? Maybe "dotnet."? I don't think it has any user impact, so feel free to ignore me. :smile:

**Comment by DustinCampbell** [comment]
:+1:

File: package.json (hunk 1 above)
Comment (caslan) [inline package.json]: Changed the prefix to dotnet

**Comment by caslan** [comment]
Addressed code review comments. @gregg-miskelly Can you merge?

**Pull request merged**